### PR TITLE
Housekeeping and minor fixes

### LINKED ISSUE #1075: Move all the data:image/png;base64 stuff in a specific file
Then, just use : 

```` javascript
this.logo = RESImages.logo;
````

instead of

```` javascript
this.logo = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADwAAAAeCAMAAABHRo19AAAACXBIWXMAAA7EAAAOxAGVKw4bAAACxFBMVEXw8/wAAAD+//8EBAQSEhIPDw/w8/v+/v4JCQkHBwcCAgKSk5W8vLz9SADz8/MtLS0iIiIcHBz/VAAYGBmRkZFkZGUkJCQVFhZiYmOZmp2QkpfQ09r9/f3n6vA5OTkvLy//TAAxMTEUFRTl5eVqa2zu8fnt7/fV19ydnqCen6Lt8Pj/TwDk5ORaWlrg4ug1NTUpKSrX19cgICDp6/J6enrFxcW1trpDQ0M7OzwnJyenp6f6TQAXFxj/WACFhojr6+uNjpBHR0cfHx+vr7GSkpJMTEwYGBg+Pj5cXF3CwsJISEj29vYQEBDe3t7+SwBmZmixsbH19fXo6OhQUFAgICJgYWXHyM3q7PTs7vW3uLvb3eKqq650dXbS09js7/aTlJY5OjmUlJeenp7r7vWWl5n8/Px4eHihoqWEhYfO0NTj5euDg4Pa3OGRkpTJy8/g4ODe4Obc3Nzv8vqjo6O1tbW3uLyrq6t1dXX5ya5/f3/5xqxZWVqKiopra2v4uJb99vLCw8fFxsouLS6Oj5Hs7OzY2t+jpKZ4eXv2tY8NDQ35WQny8vJkZGT2lWGQkJB8fHzi5OrLzNFAQUPm6O/3f0W7u7v3oXP4dTb2nXH62MX3pHb87+bn5+dWV1dvb3E0NDT4lWP3jFP4vJn2cS79+vaJioxNTU376d72f0H4Wwf2fT7759z9+fX1lmH4XAv2bSb40bheX2A6Ojr9+vj76t/9+vf76+H5XxVGRkZxcnPQ0te+vr52dnaztLfExMT2tZFYWFhSUlLV1dVwcXL52MS4uLiysrKam5rW1tZPT1CVlZWYmJiUlJRHR0ipqq0qKiqzs7P39/fq6urj4+P89fH09PT6+vo4ODjq7PNsbW4oKCh0dHTv7++3t7fk5u2IiYtFRUU3NzdPT0/Kysru7u6NjY1tbW1gYGBfX19sbGyHh4fh4eEzPXfuAAACPElEQVR4Xq3SQ9fkQBTH4bpVSdru17Zt28bYtm3btm3btm37S8yk0oteTKc7c+a3uf/Nc3JyEvT/48KF69Uhu7dk3AfaZ48PRiHgUwLdpGLdtFbecrkPOxvjuSRcmp2vaIsQt6gdLME4UtlGGs6NFW7+GIw7Qidp2BAq3KaQWg650mwC9LSs6JpRfZG03PTo32reMrmzIW3IlGaSZY/W+aCcoY/xq1SCKXAC5xAaGObkFoSmZoK3uaxqlgzL6vol3UohjIpDLWq6J4jaaNZUnsb4syMCsHU5o10q4015sZAshp2LuuCu4DSZFzJrrh0GURj3Ai8BNHrQ08TdyvZXDsDzYBD+W4OJK5bFh9nGIaRuKKTTxw5fOtJTUCtWjh3H31NQiCdOso2DiVlXSsXGDN+M6XRdnlmtmUNXYrGaLPhD3IFvoQfQrH4KkMdRsjgiK2IZXcurs4zHVvFrdSasQTaeTFu7DtPWa4yaDXSd0xh9N22mMyUVieItWwW8bfuOnbvo2r1n7779mOZ6QByHHsRChw4fsXwsz6OPsdDxE0i0kyQA20rLFIhjzuW0TVxIgpB4Z+AsBRXn1RZTdeEivXFyFbLXJTaJvmkDNJgLrly95iR3juTt9eIbyH6ucJPq2hJGQQiru63lbbriDocc6C7cu1/BgwcPH9U/4cdT9TNQIcd6/oK8fFWbg4Vev0n0I6VvkcO9A38Fq495X5T3wZkhLvAROZ6KYT59Lvvy9VvU9x8/1fW/DEygHfEbNdeCkgdk4HMAAAAASUVORK5CYII=';
````

What do you think about that ? I think it would make code easier to read.

### LINKED ISSUE #1437: possible NER / returnToPrevPage # weirdness?
http://www.reddit.com/r/RedditUpvoteDisplay/comments/2a6o0n/user_feedback/cis6k71

> when NER inserts the new content into the bottom of the siteTable div, it inserts `<div id ='page-#' ...>` and `<div id='siteTable-(#-1)1' ...>`. However, when it reloads (when pressing back button and "returnToPrevPage" enabled) it inserts `<div id ='page-#' ...> <div id='siteTable-(#)1' ...>` and the next page is inserted as `<div id ='page-(#+1)' ...> <div id='siteTable-(#)1' ...>`, resulting in two tags with the same id. Is this correct behaviour or a bug?

### PR DESCRIPTION
**Pull request by erikdesjardins**
- replace `indexOf` with `includes` or `startsWith`
- add new linting rules
- replace some `watchForElement`s with event delegation
- remove remaining base64 images in source (fixes #1075)
- fix header generation in wiki previews ([ref](https://www.reddit.com/r/RESissues/comments/3590e5/bug_incorrect_heading_layout_previews/))
- fix inconsistent NER `siteTable-{num}` IDs (fixes #1437)

### CODE REVIEW

File: lib/css/modules/_styleTweaks.scss [hunk 1]
```diff
@@ -48,6 +48,13 @@ blockquote.twitter-tweet {
 	.edited-timestamp { font-weight: bolder; }
 }
 
+.res-colorblind {
+	#mail.havemail {
```
Comment (andytuba) [inline lib/css/modules/_styleTweaks.scss]: Why dupe https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/2970/files#diff-b2aaa6b6b88ceb48617993ac868144c8R17

File: lib/modules/customToggles.js [hunk 2]
```diff
@@ -120,7 +120,9 @@ function createToggleMenuItems() {
 				anyTogglesEnabled(menuItem)
 			));
 
-		Menu.addMenuItem($toggle, function() { this::onClickToggleMenuItem(menuItemID); });
+		Menu.addMenuItem($toggle, function() {
```
Comment (andytuba) [inline lib/modules/customToggles.js]: one-liner this?

File: lib/modules/newCommentCount.js [hunk 3]
```diff
@@ -487,8 +486,6 @@ function subscribeToThread(commentID) {
 }
 
 function unsubscribeFromThread(commentID) {
```
Comment (andytuba) [inline lib/modules/newCommentCount.js]: not `commentID = currentCommentID` ?

File: lib/css/modules/_styleTweaks.scss (hunk 1 above)
Comment (erikdesjardins) [inline lib/css/modules/_styleTweaks.scss]: The other is for NER mail, but just `.havemail` here might be enough specificity for both

File: lib/modules/newCommentCount.js (hunk 3 above)
Comment (erikdesjardins) [inline lib/modules/newCommentCount.js]: It's only called in one place, the default was unnecessary

**Comment by andytuba** [comment]
lgtm

**Pull request merged**